### ISSUE TITLE
GUI almost not updating and becoming almost unresponsive with a lot of input in (a readonly) CTkTextbox

### ISSUE THREAD

**Issue by bountycortez** [TomSchimansky/CustomTkinter#1020]
Update: I forgot to mention that this does not happen, when a simple tkinter.Text wiget is used:
```
textbox = tkinter.Text(root, spacing1=0, spacing2=0, spacing3=5) 
```

Try the example: Watch the indeterminate progressbar or try to scroll.
Use different ranges instead of 8000 in the example below (on my mac pro late 2013 problems start about 4000)

code:
```
import tkinter
import customtkinter

customtkinter.set_appearance_mode("System")
customtkinter.set_default_color_theme("blue")

root=customtkinter.CTk()

root.grid_columnconfigure(0, weight=1)
root.grid_columnconfigure(1, weight=0)
root.grid_rowconfigure(0, weight=1)
root.grid_rowconfigure(1, weight=0)

textbox = customtkinter.CTkTextbox(root, spacing1=0, spacing2=0, spacing3=5) 
textbox.grid(row=0, column=0, rowspan=4, padx=(20, 20), pady=(20, 20), sticky="nsew")
for i in range(1,8000): 
    textbox.insert("end", 
        """Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.\n""" 
    * 1)
textbox.configure(state="disabled")
textbox.see("end")

progressbar = customtkinter.CTkProgressBar(root)
progressbar.grid(row=0, column=1, padx=20, pady=10)
progressbar.configure(mode="indeterminate")
progressbar.start()

root.mainloop()
```

[textbox_with_a_lot_of_input.py.txt](https://github.com/TomSchimansky/CustomTkinter/files/10373341/textbox_with_a_lot_of_input.py.txt)

**Comment by GermanYoshi** [comment]
some problem for me. After Inserting around 500 lines in a Textbox the Programm lags out..

Specs: 
i7-9700k
32gb Ram

**Comment by TomSchimansky** [comment]
I already noticed this, I will have a look at this problem.

**Comment by bountycortez** [comment]
by the way: great job, respect!

Tom Schimansky ***@***.***> schrieb am Di. 10. Jän. 2023 um
12:00:

**Comment by Akascape** [comment]
@bountycortez For now, you can use the normal tkinter text widget with customization:
https://github.com/TomSchimansky/CustomTkinter/issues/897

**Comment by bountycortez** [comment]
> 

thank you! very nice, was building the same solution (half way), perfect!

**Comment by TomSchimansky** [comment]
should be fixed now with version 5.0.5

**Comment by bountycortez** [comment]
can confirm, thank you very much.
